features/unspecified-apis.yml

```diff
@@ -0,0 +1,27 @@
+name: Unspecified APIs
+description: Some browsers implement unspecified APIs. This is an artificial feature that contains references to browser compatibility data entries for those additions.
+spec: link to our own documentation explaining ourselves in more detail
+discouraged:
+  according_to:
+    - TODO link to our own documentation explaining ourselves in more detail
```

Comment:
Can't land without this, right?

Reply:
Correct. I've added one with 83a63334806718ea7ef9ea15d87df05db0a4c7c1.